apps.toml

```
@@ -611,7 +611,7 @@ url = "https://github.com/YunoHost-Apps/digiwords_ynh"
 
 [discourse]
 category = "communication"
-level = 8
+level = 1
```

```suggestion
level = 8
```

Tests fail due to timeouts. Last successful test on same package version: https://ci-apps.yunohost.org/ci/job/13442